SpatialGDK/Source/SpatialGDK/Private/SpatialView/SubView.cpp

```diff
@@ -0,0 +1,209 @@
+﻿// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialView/SubView.h"
+
+#include "Utils/ComponentFactory.h"
+
+namespace SpatialGDK
+{
+FSubView::FSubView(const Worker_ComponentId& TagComponentId, FFilterPredicate Filter, const EntityView* View, FDispatcher& Dispatcher,
+				   const TArray<FDispatcherRefreshCallback>& DispatcherRefreshCallbacks)
```

Comment:
Agree that the dispatcher callback interface is fairly complex and difficult to parse. However it's functional and well contained, so should be relatively simple enough to rework if and when we feel it's necessary.